docs/build.gradle

```diff
@@ -52,8 +52,8 @@ integTestCluster {
   configFile 'userdict_ja.txt'
   configFile 'userdict_ko.txt'
   configFile 'KeywordTokenizer.rbbi'
-  extraConfigFile 'hunspell/en_US/en_US.aff', '../server/src/test/resources/indices/analyze/conf_dir/hunspell/en_US/en_US.aff'
-  extraConfigFile 'hunspell/en_US/en_US.dic', '../server/src/test/resources/indices/analyze/conf_dir/hunspell/en_US/en_US.dic'
+  extraConfigFile 'hunspell/en_US/en_US.aff', file('../server/src/test/resources/indices/analyze/conf_dir/hunspell/en_US/en_US.aff')
```

Comment:
At the very least if we are going to do cross-project resource sharing we should avoid using relative paths. Let's do `project(':server').file()` instead.